### DISCUSSION TITLE
Unblock Crosh Via terminal

### DISCUSSION THREAD

**Discussion by RacoonyCoon** [3kh0/ext-remover#1383] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I tried doing this and it worked however it just says Loading pnacl program... is there anything I can do?

**Comment by schoolexploitkid** [comment]
this exploit exists???

**Comment by akabutnicer** [comment, marked as answer]
yeah so this is intended behavior for those with SSH blocked (as per your administrator)
